challenge_10/python/shaowen/unit_test.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jan 11 09:21:20 2017
+
+@author: Shaowen Liu
+"""
+import unittest
+from solution import solution
+
+class solution_test(unittest.TestCase):
+    
+    def test_case1(self):
+        strr = '{{{}}}'
+        self.assertEqual(solution(strr), True)
```

Comment:
Instead of using `assertEqual` to test True / False conditions, use `assertTrue` and `assertFalse`.